src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -54,7 +54,15 @@ See the accompanying LICENSE file for applicable license.
     
     <basename property="dita.input.filename" file="${args.input}" />
     <basename property="dita.map.filename.root" file="${dita.input.filename}" suffix=".ditamap" />
-    <basename property="dita.topic.filename.root" file="${dita.input.filename}" suffix=".dita" />
+    <condition property="dita.topic.filename.extension" value=".xml">
```

Comment:
Without getting into ant-contrib, there isn't really an easy way to the extension without using regex.